misc/ls-extensions/modules/metadata-to-record-generator/src/main/java/io/ballerina/generators/MetadataToRecordClientCapabilities.java

```diff
@@ -0,0 +1,32 @@
+package io.ballerina.generators;
+
+import org.ballerinalang.langserver.commons.registration.BallerinaClientCapability;
+
+/**
+ * Client capabilities for the custom records service.
+ * The methods which are exposed to the client from the extension.
+ */
+public class MetadataToRecordClientCapabilities extends BallerinaClientCapability {
+    private boolean getResults;
+    private boolean getRemoteFunctionCalls;
+
+    public MetadataToRecordClientCapabilities() {
+        super(MetadataToRecordConstants.CAPABILITY_NAME);
+    }
+
+    public boolean isGetResults() {
+        return getResults;
+    }
+
+    public void setGetResults(boolean getResults) {
+        this.getResults = getResults;
+    }
```

Comment:
Let's regenerate the getters and setters after adding a proper name for the boolean variables.